Description: For clarity, mention that possible attribute-name are
 defined in RFC2911.
Author: Helge Kreutzmann <debian@helgefjell.de>
Bug-Debian: http://bugs.debian.org/711709
Last-Update: 2013-06-09
--- a/man/ipptoolfile.man
+++ b/man/ipptoolfile.man
@@ -144,7 +144,7 @@
 .TP 5
 ATTR tag attribute-name value(s)
 Adds an attribute to the test request. Values are separated by the comma (",")
-character \- escape commas using the "\" character.
+character \- escape commas using the "\\" character. Possible attribute-name are defined in RFC2911.
 .TP 5
 ATTR collection attribute-name { MEMBER tag member-name value(s) ... } [ ... { ... } ]
 Adds a collection attribute to the test request. Member attributes follow the
@@ -566,6 +566,8 @@
 \fIipptool(1)\fR,
 .br
 http://localhost:631/help
+.br
+RFC2911 http://tools.ietf.org/html/rfc2911
 
 .SH COPYRIGHT
 Copyright 2007-2013 by Apple Inc.
